controller/src/main.cpp

```diff
@@ -60,6 +61,31 @@ static ControllerStatus controller_status = ControllerStatus_init_zero;
 // Last-received status from the GUI.
 static GuiStatus gui_status = GuiStatus_init_zero;
 
+// desired actuators state
+static ActuatorsState actuators_state;
```

Comment:
No need for this to be a global variable (right now)?  We recreate it on every iteration of the loop.